napari/layers/labels/labels.py

```diff
@@ -340,6 +352,45 @@ def selected_label(self, selected_label):
         self._selected_color = self.get_color(selected_label)
         self.events.selected_label()
 
+    @property
+    def enable_custom_color_dict(self):
```

Comment:
This should become `color_mode` which I think for now should either be
`RANDOM` or `DICT`, but in the future could be based off of an attribute in the properties dict. I do actually wonder @kevinyamauchi if since #1281 has merged we want to force this PR to use the properties dict rather than add an additional `color_dict` (in which case we might want to drop that from the API too)

Reply:
if we make it another drop down, the panel seems crowded, wonder we can move things around?

Reply:
I'd put it on a new line underneath blending. For now we'd just have those two options, but we know we want to add a third.

You might want to play with the `add_vectors_color_by_angle.py` example and see how the property `color mode` works there to get a sense of where this might go
<img width="1200" alt="Screen Shot 2020-06-08 at 8 04 50 PM" src="https://user-images.githubusercontent.com/6531703/84101744-8434ef80-a9c3-11ea-976e-5e5b73eb06c9.png">